src/main/scala/org/scalameter/BasePerformanceTest.scala

```diff
@@ -2,13 +2,17 @@ package org.scalameter
 
 
 import java.util.Date
+import org.scalameter.picklers.Pickler
 import org.scalameter.utils.Tree
+import scala.language.existentials
 import scala.util.DynamicVariable
 
 
 
 abstract class BasePerformanceTest {
 
+  type Result
```

Comment:
OK, but won't it implicate that we won't be able to provide default measurer, reporter and aggregator in the Bench implementation? Or other way, if we provide them and stick to Double user won't be able to use other measurers instead.